gym/vector/utils/info_strategies.py

```diff
@@ -59,21 +62,37 @@ class BraxVecEnvInfoStrategy(VecEnvInfoStrategy):
 
     Example with 3 environments in which only the last has the `terminal_observation` info:
         {
```

Comment:
Could you make this an actual docstring code example. See TransformObservation wrapper for an example, you should just need to put a seperating line, >>> for the first line and ... for the rest